test/test_sparse_csr.py

```diff
@@ -729,37 +729,35 @@ def _generate_invalid_input(self, layout):
                torch.tensor([0, 1, 0, 2]),
                values([1, 2, 3, 4]),
                shape((2, 3)),
-               r'compressed_indices\[0\] \(=1\) == 0 is unsatisfied')
+               r'`c{row|col}_indices[..., 0] == 0` is not satisfied.')
```

Comment:
nit: I think the original messaging (`compressed_indices` / `plain_indices`) was clearer. The `c{row|col}_indices`/ `{row|col}_indices` might be bit more difficult for the user to parse. As the compressed layouts are very similar, unified documentation could re-use the `compressed_` and `plain_` terminology as well.

Reply:
Totally agree, and I have been waiting for such a comment, provided this is something we agree upon, cc @cpuhrsch .

Reply:
I think either is fine. Also cc @pearu

Reply:
I'd prefer the original messages as `c{row|col}_indices` is rather cryptic and contains redundant/ambiguous information (such as `ccol` for CSR layout).

Notice that these error messages are user-facing and for a high-quality UI, such error messages should be clear, minimal, and exact.

Reply:
Btw, the original error messages would contain `crow_indices` for CSR/BSR layout and `ccol_indices` for CSC/BSC layout, not `compressed_indices` that is specified here only for testing and is replaced by
https://github.com/pytorch/pytorch/blob/fc10a6372736dd8bac1a894a92beebbafee74f01/test/test_sparse_csr.py#L794-L805
with the actual names.

Reply:
Actual names are not possible, unless we construct these strings at compile time, otherwise huge perf hit on CUDA. If only either of layout had something `constexpr` to be usable as a template parameter to be able to generate such strings.

Reply:
Yes, that is why I think this changeset represents the case where the code is over-optimized. I think this optimization is not needed. See issue 2 in https://github.com/pytorch/pytorch/pull/81108#pullrequestreview-1033358399